src/commonMain/kotlin/com/github/doyaaaaaken/kotlincsv/client/CsvWriterState.kt

```diff
@@ -0,0 +1,31 @@
+package com.github.doyaaaaaken.kotlincsv.client
+
+/**
+ * CSV writer state and handler
+ * @author blacmko18
+ */
+sealed class CsvWriterState(val wroteFirstLine: Boolean, val wroteLastLineTerminator: Boolean)
+
+object DefaultState: CsvWriterState(wroteFirstLine = false, wroteLastLineTerminator =  false)
+object HasNotWroteLastLineTerminator: CsvWriterState(true, wroteLastLineTerminator = false)
+object HasWroteLastLineTerminator: CsvWriterState(true, wroteLastLineTerminator = true)
```

Comment:
The word `LastLine` is misleading. 
`LineEnd` would be more appropriate.

Reply:
change at e99bb94